lib/licenses.nix

```diff
@@ -1212,6 +1212,12 @@ lib.mapAttrs mkLicense ({
     fullName = "TCL/TK License";
   };
 
+  tost = {
+    shortName = "TSL";
```

Comment:
Is this a short name for which there is precedent? If not, I'd rather just not declare it.

  ```suggestion
```

Reply:
yes, also from https://forum.aousd.org/t/upcoming-openusd-license-update/1561

Reply:
Those 3 letters do not appear on that page in that order.

Reply:
Oh, yes I see that now. This is a typo I think and should either be "TOST" or removed

Reply:
This defaults to the license attr name, so simply remove it.